samza-shell/src/main/bash/run-class.sh

```diff
@@ -44,10 +44,12 @@ DEFAULT_LOG4J_FILE=$base_dir/lib/log4j.xml
 BASE_LIB_DIR="$base_dir/lib"
 # JOB_LIB_DIR will be set for yarn container in ContainerUtil.java
 # for others we set it to home_dir/lib
-JOB_LIB_DIR="${JOB_LIB_DIR:-$home_dir/lib}"
+JOB_LIB_DIR="${JOB_LIB_DIR:-$base_dir/lib}"
```

Comment:
Not related to your changes, but line number 27, 28 are No-op and can be safely deleted. Please delete them and test it.

Reply:
Will open a cleaning Jira